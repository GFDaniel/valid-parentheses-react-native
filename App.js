import { StatusBar } from 'expo-status-bar';
import { StyleSheet, View } from 'react-native';
import  Validator  from './src/components/Validator';

export default function App() {
  return (
    <View style={styles.container}>

      <StatusBar style="auto" />
      <Validator/>
      
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
