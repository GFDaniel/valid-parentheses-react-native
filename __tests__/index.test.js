import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';
describe('Render App', () => {
    it('renders correctly', () => {
        const component = renderer.create(<App />).toJSON();
        expect(component).toBeDefined(); 
        expect(component).toMatchSnapshot();
    });
});
