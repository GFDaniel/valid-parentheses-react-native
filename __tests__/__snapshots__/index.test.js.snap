// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      {
        "paddingHorizontal": 20,
        "paddingTop": 100,
      }
    }
  >
    <Text
      className="unbalancedBrackets"
      style={
        {
          "color": "red",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 50,
          "textAlign": "center",
        }
      }
    >
      Los parentesis, Corchetes, etc no estan equilibrados
    </Text>
    <Text
      style={
        {
          "color": "grey",
          "fontSize": 14,
          "marginBottom": 10,
        }
      }
    >
      Ingresa tu codigo:
    </Text>
    <TextInput
      multiline={true}
      numberOfLines={10}
      onChangeText={[Function]}
      style={
        {
          "flex": 1,
          "height": 200,
          "textAlignVertical": "top",
        }
      }
      value="const addOne = value => { return value + 1; }};"
      variant="standard"
    />
  </View>
</View>
`;
